pkgs/top-level/all-packages.nix

```diff
@@ -33912,8 +33912,6 @@ with pkgs;
 
   insync = callPackage ../applications/networking/insync { };
 
-  insync-v3 = libsForQt5.callPackage ../applications/networking/insync/v3.nix { };
```

Comment:
we should add an alias for this

Reply:
Is it still good to keep this -v3 alias though? maybe time to deprecate it if there is no intention to keep maintaining multiple versions going forward?

Reply:
see https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/submitting-changes.chapter.md#:~:text=Add%20an%20alias%20for%20the%20package%20name%20in%20pkgs/top%2Dlevel/aliases.nix and https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix#L3-L18

Reply:
Thank you for pointing this out, I amended the commits!